### DISCUSSION TITLE
Example code for editor on start page

### DISCUSSION THREAD

**Discussion by pReya** [ueberdosis/tiptap#2511] [General Questions] [answered]
When visiting www.tiptap.dev there is one Tip Tap example on the middle of the home page. I'd really like to see the code for this editor example (because it uses an actual toolbar with icons compared to all the other examples). I couldn't find it anywhere in the repo. Is this available anywhere? Appreciate your help...

![Screenshot 2022-02-08 at 16 11 28](https://user-images.githubusercontent.com/4677417/153016013-8de7ead4-bb42-47f6-9361-bd81861b7d5d.png)

**Comment by philippkuehn** [comment, marked as answer]
It’s the collab example: https://tiptap.dev/examples/collaborative-editing

**Reply by pReya** [reply]
Thanks so much!
